### DISCUSSION TITLE
Token store changed to premium only?

### DISCUSSION THREAD

**Discussion by MemenTheMeme** [HelixNebulaStudio/RiseOfTheDead#394] [General]
Why did the token store get changed to premium only after the event started? Cant spend 200 tokens now even though I bought other items from the store earlier :\

**Comment by GregTheBestest** [comment]
I believe it was intended to be premium since very beginning as its window in pass was yellow even in dev branch.

**Comment by MemenTheMeme** [comment]
if that was the case, why would it allow non premium members to trade in items for tokens to begin with?

**Reply by GregTheBestest** [reply]
Because it would probably take some time to make the unlockables exchange system only for premium members, and not everybody.

**Reply by Leroy8888** [reply]
It already was intended to be premium only from the start, but it appears that was a bug. So people without premium were lucky for the time being.

A change was made already however and it is now usable by both premium and non premium for the duration of the event. See #395.
